### ISSUE TITLE
[Test Failure][Dotnet List Package]An error occur when executing dotnet list package in a project with multiple TFMs

### ISSUE THREAD

**Issue by v-luzh** [NuGet/Home#7713]
## Details about Problem
OS Version: Windows 10 RS4_Release 17134.1.180410-1804
Dotnet SDK: 3.0.100-preview-010140
## Detailed repro steps so we can see the same problem
1. Create a powershell script with below content.
```
"Getting latest .NET Core CLI"
if (!(Test-Path .\cli)) {
    New-Item .\cli\ -ItemType Directory | Out-Null
}
if (!(Test-Path .\cli\dotnet-install.ps1)) {
    Invoke-WebRequest https://raw.githubusercontent.com/dotnet/cli/master/scripts/obtain/dotnet-install.ps1 -OutFile .\cli\dotnet-install.ps1
}
.\cli\dotnet-install -i cli -c master
 
"Creating test package projects"
.\cli\dotnet new classlib -n MyLib
.\cli\dotnet new classlib -n MyDep
.\cli\dotnet add MyLib/MyLib.csproj reference MyDep/MyDep.csproj
.\cli\dotnet new sln -n test
.\cli\dotnet sln add MyLib/MyLib.csproj
.\cli\dotnet sln add MyDep/MyDep.csproj
 
.\cli\dotnet build
 
"Creating packages"
.\cli\dotnet msbuild /t:pack /p:version=1.0.1
.\cli\dotnet msbuild /t:pack /p:version=1.0.2
.\cli\dotnet msbuild /t:pack /p:version=1.0.3-beta
.\cli\dotnet msbuild /t:pack /p:version=1.1.0
.\cli\dotnet msbuild /t:pack /p:version=1.2.0-beta
.\cli\dotnet msbuild /t:pack /p:version=2.0.0
.\cli\dotnet msbuild /t:pack /p:version=3.0.0-beta
 
"Creating nuget.config"
New-Item .\localSource\ -ItemType Directory | Out-Null
.\cli\dotnet new nugetconfig
$xml = [xml](Get-Content .\nuget.config) 
$sources = $xml.SelectSingleNode('/configuration/packageSources')
$sources.RemoveAll()
$source = $xml.CreateElement('add')
$source.SetAttribute('key', 'localSource') | Out-Null
$source.SetAttribute('value', 'localSource') | Out-Null
$sources.AppendChild($source)
$source = $xml.CreateElement('add')
if (Test-Path .\cli\sdk\NuGetFallbackFolder) {
    $source.SetAttribute('key', 'dotnetFallbackFolder') | Out-Null
    $source.SetAttribute('value', 'cli/sdk/NuGetFallbackFolder') | Out-Null
    $sources.AppendChild($source)
}
$config = $xml.CreateElement('config')
$xml.DocumentElement.AppendChild($config) | Out-Null
$globalPackagesFolder = $xml.CreateElement('add')
$globalPackagesFolder.SetAttribute('key', 'globalPackagesFolder')
$globalPackagesFolder.SetAttribute('value', 'globalPackages')
$config.AppendChild($globalPackagesFolder) | Out-Null
$xml.Save((Join-Path (Get-Location) nuget.config))
Copy-Item *\bin\Debug\*.nupkg .\localSource\ | Out-Null
 
"Creating test program"
.\cli\dotnet new console -n MyProg
.\cli\dotnet add MyProg\MyProg.csproj package MyLib -v 1.0.0
.\cli\dotnet restore MyProg\MyProg.csproj
 
.\cli\dotnet list MyProg\MyProg.csproj package --outdated --include-transitive --highest-patch
"latest version of MyLib and MyDep above should be 1.0.2"
Pause
 
.\cli\dotnet list MyProg\MyProg.csproj package --outdated --include-transitive --highest-patch --include-prerelease
"latest version of MyLib and MyDep above should be 1.0.3-beta"
Pause
 
.\cli\dotnet list MyProg\MyProg.csproj package --outdated --include-transitive --highest-minor
"latest version of MyLib and MyDep above should be 1.1.0"
Pause
 
.\cli\dotnet list MyProg\MyProg.csproj package --outdated --include-transitive --highest-minor --include-prerelease
"latest version of MyLib and MyDep above should be 1.2.0-beta"
Pause
 
.\cli\dotnet list MyProg\MyProg.csproj package --outdated --include-transitive
"latest version of MyLib and MyDep above should be 2.0.0"
Pause
 
.\cli\dotnet list MyProg\MyProg.csproj package --outdated --include-transitive --include-prerelease
"latest version of MyLib and MyDep above should be 3.0.0-beta"
Pause
 
"testing complete with the following version of the .NET Core CLI/SDK"
.\cli\dotnet --version

.\cli\dotnet new classlib -n MultiTargetTest
$xml = @'
<Project Sdk="Microsoft.NET.Sdk">
 
  <PropertyGroup>
    <TargetFrameworks>netstandard1.3;netstandard2.0</TargetFrameworks>
  </PropertyGroup>
 
  <ItemGroup>
    <PackageReference Include="Newtonsoft.Json" Version="11.0.1" />
  </ItemGroup>
 
  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard1.3'">
    <PackageReference Include="Moq" Version="4.10.0" />
  </ItemGroup>
 
  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
    <PackageReference Include="xunit" Version="2.4.0" />
  </ItemGroup>
 
</Project>
'@
Out-File -FilePath .\MultiTargetTest\MultiTargetTest.csproj -InputObject $xml
.\cli\dotnet restore MultiTargetTest\MultiTargetTest.csproj
.\cli\dotnet list MultiTargetTest\MultiTargetTest.csproj package
"The output above should have two target frameworks, netstandard1.3 and netstandard2.0"
"Moq should be listed under netstandard1.3, but not netstandard2.0"
"xunit should be listed under netstandard2.0, but not netstandard1.3"
"Newtonsoft.Json should be listed under both"
pause
```
2. Execute the script in PowerShell command window. (Press Enter when meeting every pause)
## Expected
The output of last command line (.\cli\dotnet list MultiTargetTest\MultiTargetTest.csproj package) should have two target frameworks, netstandard1.3 and netstandard2.0. Moq should be listed under netstandard1.3, but not netstandard2.0, xunit should be listed under netstandard2.0, but not netstandard1.3.Newtonsoft.Json should be listed under both.
## Actual
An error occur when executing dotnet list package in a project with multiple TFMs as below.
**Error message:**
error: A numeric comparison was attempted on "$(_TargetFrameworkVersionWithoutV)" that evaluates to "" instead of a number, in condition "'$(_TargetFrameworkVersionWithoutV)' < '3.0'".  C:\Dotnet\cli\sdk\3.0.100-preview-010140\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.DefaultItems.props
![erroroccur](https://user-images.githubusercontent.com/20632625/51518726-d078a180-1e59-11e9-92cd-722511f25b59.png)
## Note
1.It repro on Dotnet SDK: 3.0.100-preview-010137
2.It didn’t repro on Dotnet SDK: 3.0.100-preview-010086

**Comment by v-luzh** [comment]
The bug is fixed on Dotnet SDK 3.0.100-preview-010185 / 3.0.100-preview-010206.

**Comment by rrelyea** [comment]
closing...likely was an external issue. don't think we've fixed anything in this area to make it go way.
